swagger.json

```diff
@@ -3369,6 +3369,17 @@
               "items": {
                 "type": "object",
                 "properties": {
+                  "id": {
+                    "type": "integer",
+                      "format": "int64",
+                      "description": "The id of this DVIR record.",
+                      "example": 19
+                  },
+                  "timeMs": {
+                    "type": "integer",
```

Comment:
Nit: could we also include `"format": "int64"` here?

Reply:
Sure!

Reply:
fixed!